build_tools/benchmarks/common/benchmark_config_test.py

```diff
@@ -11,16 +11,16 @@
 import tempfile
 import os
 
-from common.common_arguments import build_common_argument_parser
-from common.benchmark_config import BenchmarkConfig, TraceCaptureConfig
+from common import benchmark_config, common_arguments
 
 
 class BenchmarkConfigTest(unittest.TestCase):
 
   def setUp(self):
     self.build_dir = tempfile.TemporaryDirectory()
-    self.tmp_dir = tempfile.TemporaryDirectory()
     self.build_dir_path = pathlib.Path(self.build_dir.name).resolve()
+    self.tmp_dir = tempfile.TemporaryDirectory()
+    self.tmp_dir_path = pathlib.Path(self.tmp_dir.name).resolve()
```

Comment:
This is basically now an implementation detail of the setup. I think you can mark it as private and keep the shorter `tmp_dir` name for the pathlib version

```suggestion
    self._tmp_dir_manager = tempfile.TemporaryDirectory()
    self.tmp_dir = pathlib.Path(self._tmp_dir_manager.name).resolve()
```

Reply:
Done.